db/tests/tables/operations/test_infer_types.py

```diff
@@ -25,6 +26,7 @@
     (String, ["t", "false", "2", "0"], VARCHAR),
     (String, ["a", "cat", "mat", "bat"], VARCHAR),
     (String, ["2", "1", "0", "0"], NUMERIC),
+    (String, ["$2", "$1", "$0"], MONEY),
```

Comment:
Be aware, this test assumes the test DB is in a locale that uses `'$'` for the currency symbol.  Not a problem in the current docker setup, but somehow disconcerting.

Reply:
I made changes to the sqlalchemy test engine to use an `en_US.UTF-8` locale, but it would still throw error for api test that uses a different engine. Should I document somewhere that our test cases are written with `en_US.UTF-8` in mind?

Reply:
I think it's okay as long as the engine used for these tests specifies the appropriate locale.